### ISSUE TITLE
Question/Suggestion

### ISSUE THREAD

**Issue by TamasAYM** [mrgregory/scrum-workflow#1]
I think around ToDo and In Progress there should be a state defined, either with the impediment jira flag to indicate when a task is blocked (then stay in progress) or create a new state such as "waiting for feedback" that could be used when specification is unclear or you are waiting for another party to provide information before you can continue working on the ticket.

The arrow from Done should be from QA Review to ToDo, no?

For resolution I would add "Invalid" as well, this will be super important once we have clear documentation of features and acceptance criteria.

I think the hourly estimation should NOT happen DURING the sprint planning meeting, but should be done before, and the meeting we should just review the numbers.

**Comment by mrgregory** [comment]
About On Hold status what you mention, I think we don't need it, because instead of On Hold it could go back to To Do and assign it to the responsible person.

Why we need to move a ticket to QA Review from Done?

We could add separate Invalid resolution, but I think it's the Won't fix then specify the reason.

We should assign the tickets on planning then we could estimate the time based on that person.

**Comment by mrgregory** [comment]
As we discussed:
- we should replace the Cannot Reproduce with Invalid for more generic propose
- we can assign the ticket on pre-planning then the individuals can investigate for the hours estimation
